docs/config.md

```diff
@@ -8,8 +8,8 @@ for more information about containerd config.
 
 The explanation and default value of each configuration item are as follows:
 ```toml
```

Comment:
Add `version = 2`. Version 2 is not the default, we need to explicitly specify it.

See https://github.com/containerd/containerd/pull/3337

Reply:
version 1 is the default if the field is not specified...

Reply:
Here's my suggested text: 

The version field in the config file specifies the config's version. If no version number is specified inside
 the config file then it is assumed to be a version 1 config and parsed as such. Use version = 2 to enable/access to new/current config fields.

Reply:
Done.